app/app.js

```diff
@@ -38,6 +38,8 @@ var backendUrl = window.ushahidi.backendUrl = (window.ushahidi.backendUrl || BAC
     intercomAppId = window.ushahidi.intercomAppId = window.ushahidi.intercomAppId || '',
     appStoreId = window.ushahidi.appStoreId = window.ushahidi.appStoreId || '',
     apiUrl = window.ushahidi.apiUrl = backendUrl + '/api/v3',
+    platform_websocket_redis_adpater_url = window.ushahidi.platform_websocket_redis_adpater_url || '',
+    platform_websocket_redis_adpater_port = window.ushahidi.platform_websocket_redis_adpater_port || '',
```

Comment:
It will work regardless, it's agnostic of redis, there is a feature config var on the API that the client has access to. It won't try to start a websocket connection unless redis is enabled.